assets/src/application/ui/layout/espressoTable/TableBody.tsx

```diff
@@ -96,9 +96,8 @@ const TableBody: React.FC<TableBodyProps> = ({
 				{({ innerRef, droppableProps, placeholder }, { isDraggingOver }) => (
 					<tbody
 						ref={innerRef}
-						className={htmlClass}
+						className={className}
 						style={{
-							...droppableProps.style,
```

Comment:
this has been removed based on this:
![image](https://user-images.githubusercontent.com/1477453/75721778-9e36a600-5ce1-11ea-9a41-080c7142459c.png)

Reply:
and this: https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/api/droppable.md#droppable-props